app/src/sharedTest/java/org/oppia/android/app/devoptions/MarkTopicsCompletedActivityTest.kt

```diff
@@ -143,390 +105,12 @@ class MarkTopicsCompletedActivityTest {
     assertThat(title).isEqualTo(context.getString(R.string.mark_topics_completed_activity_title))
```

Comment:
It might be worth having a few basic tests to verify that the correct UI is showing in the activity in both portrait & landscape. I don't think it pays to go into the same level of detail as the fragment tests, but something as simple has verifying the correct headers are being shown could be useful (consider that MarkTopicsCompletedActivity could never add the fragment & this test suite + the fragment test suite could still pass, assuming the latter is using a test activity as suggested in a later comment).

Reply:
Hmm. But the activity only has the `FrameLayout` and even the toolbar is part of the fragment. So, I'm not sure what additional tests we can add here. cc: @rt4914

Reply:
I have added a test (and its config change version) to test if the fragment is displayed or not. Will this be enough @BenHenning?

Reply:
Hmm. If you remove the code from MarkTopicsCompletedActivityPresenter that adds the fragment, do these tests still pass?

Reply:
No, the tests fail if I remove that code from `MarkTopicsCompletedActivityPresenter`.

Reply:
Just to double check: these specific tests in MarkTopicsCompletedActivityTest fail if that code is removed? If that's the case, then I think the tests you added here are sufficient.

Reply:
Yes.